README.md

```diff
@@ -1,6 +1,21 @@
 # Chain Selectors
 
-CCIP uses its own set of chain selectors represented by uint64 to identify blockchains. This repository contains a
+CCIP uses its own set of chain selectors represented by uint64 to identify blockchains. It is a random
+integer generated as follows:
+```python3
```

Comment:
Is it generated then validated against existing selectors? I presume it must be unique, so if we were in a weird universe where we had a dupe, we'd fail and try again, yeah?